Login/Signup, Contact, 404, and CSS changes

farrahirodriguez opened the pull request:
Deleted default files that came with the repo - auth.css & reset.css

New Files include:
- contact.html & contact.css: feedback/contact form
    - I found this site that will allow us to have submissions be sent directly to an email but it won't work yet as the site isn't up
    - https://formsubmit.co/

- login.html, login.css, and login.js: for the signup/login page
- 404.css: styling for the error page

Fixes:
- Mostly changes to the css including moving header and footer css to base.css
- customizing the signup/login css as well as contact form
- adding more info to the about page
- notably I moved the "Get Recommendations" button to the header as I felt it was more visible and kept the flow of the page but let me know what you think. 

Still Working On:
- Fixing footer problem
- Having more movies than spiderman lol

**About Page**
![image](https://github.com/jonathanhouge/ReeldIn/assets/102585852/9cb959c3-f2ac-474d-82fc-49153879fc50)

**Contact Page**
![image](https://github.com/jonathanhouge/ReeldIn/assets/102585852/822008c4-4468-4465-aa23-3bb1d4fab221)

**404**
![image](https://github.com/jonathanhouge/ReeldIn/assets/102585852/65f753f4-90db-4f6b-b2e5-829a49f2d142)

**Login**
![image](https://github.com/jonathanhouge/ReeldIn/assets/102585852/c5888a6c-1e28-44a1-ab15-5d81d64062f0)

farrahirodriguez committed:
Login, Contact page, 404 styling, and fixes

- Deleted default files that came with the repo - auth.css & reset.css

New Files include:
- contact.html & contact.css: feedback/contact form
   - I found this site that will allow us to have submissions be sent directly to an email but it won't work yet as the site isn't up

- login.html, login.css, and login.js: for the signup/login page
- 404.css: styling for the error page

farrahirodriguez committed:
deleted default icon

farrahirodriguez committed:
slight fix for 404 styling

jonathanhouge commented on accounts/static/accounts/css/login.css:
```diff
@@ -0,0 +1,300 @@
+  margin: 20px 0;
+}
+
+.social-icons a {
```
is this class still needed? know we talked about not allowed for sign in through external means and/or not having social media accounts for the website

jonathanhouge commented on accounts/static/accounts/css/login.css:
```diff
@@ -0,0 +1,300 @@
+  transform: translateX(200%);
+}
+
+/* ADAPTIVE  for mobile devices */
```
up to you if you want to keep this - i think it's kind of weird to have one bit capable of being displayed on a mobile device and nothing else, ya know?

jonathanhouge commented on accounts/templates/accounts/base.html:
```diff
@@ -1,19 +1,54 @@
-{% load static %}
-{% load static tailwind_tags %}
+<!-- 
+The following files have been adapted from "Animnated-LoginForm" by Rahman Samadzade
```
'Animnated'

jonathanhouge commented on accounts/templates/accounts/base.html:
i would say it's worth bringing in the license from this repo (like i did with 'django-template-license' or whatever it is) and including the usage in our readme

jonathanhouge commented on accounts/templates/accounts/base.html:
```diff
@@ -1,19 +1,54 @@
+
+</html>
+
+<!-- 
```
delete this and just include the license itself

jonathanhouge commented on accounts/views.py:
kind of a nasty git dif to check but i think it looks good! just mentioning my acknowledgement and potential lack of catching something.
assuming you've tested it so i trust it :)

farrahirodriguez replied:
Login in the only one doing anything at the moment and it works for me so it should be good! They each have some functions in accounts/views.py that came with the repo but it's not being used yet.

jonathanhouge commented on landing_page/templates/landing_page/about.html:
```diff
@@ -19,6 +20,33 @@ <h2>What is ReeldIn?</h2>
+    <br />
+    <p>
+      A: To get recommendations, click the "Get Recommendations" button on the
+      home page. You'll be prompted to input your favorite movies, and ReeldIn
```
need to revisit this as the 'get recommendations' button, by default, leads to twenty questions.

i think it would be worth exploring the idea of 'do you want to query our database' or 'do you want to watch something based on what you've seen', buttons for that (like one on either side of that home page) or a different page that then redirects you from there

farrahirodriguez replied:
Totally, we can change things around. I think we can have another page for getting recommendations and/or include those buttons in the user's profile so that they can quickly transition from viewing movies they've added to their lists to checking out more movies.

jonathanhouge commented on landing_page/templates/landing_page/about.html:
```diff
@@ -19,6 +20,33 @@ <h2>What is ReeldIn?</h2>
+    <h3>Q: How does ReeldIn work? <br /></h3>
+    <br />
+    <p>
+      A: ReeldIn uses a combination of user input and machine learning to
```
need to revisit this as i don't think we have any machine learning planned?
at least i don't...

farrahirodriguez replied:
The filtering I'll be doing for the recommendation system for imported user data uses Machine learning algorithms

jonathanhouge commented on landing_page/templates/landing_page/contact.html:
same thing with the login stuff - make a copy of the license and throw it in as a separate file and mention it in the readme.
in fact, we can have a folder called like 'external-licenses'

jonathanhouge commented on landing_page/templates/landing_page/index.html:
```diff
@@ -3,10 +3,9 @@
+<div class="content_div">
   <div id="title_div">
     <h2>Latest Recommendations...</h2>
-    <button id="get_recommendations_btm">Get Recommendations</button>
```
why is this button getting axed? i'm using it in my branch.
if there was relocation i missed, let me know so i can place the functionality elsewhere!

farrahirodriguez replied:
Yeah, the button is being moved to the header next to the login button/ profile pic when users are logged in. I thought that it looked better and can be seen on all pages now (without repeated code).

jonathanhouge reviewed (changes requested):
whoa, that's a lot of changes! i definitely think our prs should be smaller than this, not only for ease of reviewing, but also so changes are shared more frequently.

awesome stuff overall, just some comments and the like to address :)

farrahirodriguez committed:
Implemented Requested Changes

- removed social-icons & mobile css
- added external licenses folder & moved licenses
- added references of outside sources to readme
- spelling fix
- changed answer to about page question (left open to whatever we choose)

jonathanhouge reviewed (dismissed):
radical! :)

NerdRez reviewed (dismissed):
Looks good! I also agree that a smaller pull request would be nicer.

farrahirodriguez committed:
Merge branch 'master' into farrah-frontend
